spec/rubocop/cop/lint/literal_as_condition_spec.rb

```diff
@@ -549,4 +549,32 @@
       end until false
     RUBY
   end
+
+  %w[return break next].each do |keyword|
```

Comment:
Can you separate the examples into three different ones instead of using `each`?

Reply:
Sure. I tweaked examples to remove the block when not necessary for it to be valid syntax for `return`.